pkg/cmd/workspace/logs.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package workspace
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/daytonaio/daytona/cmd/daytona/config"
+	"github.com/daytonaio/daytona/internal/util"
+	apiclient_util "github.com/daytonaio/daytona/internal/util/apiclient"
+	logs_view "github.com/daytonaio/daytona/pkg/views/logs"
+	"github.com/daytonaio/daytona/pkg/views/workspace/selection"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+)
+
+var followFlag bool
+var retryFlag bool
+
+var LogsCmd = &cobra.Command{
+	Use:     "logs [WORKSPACE] [PROJECT]",
```

Comment:
We should somehow allow the user to get all logs of a workspace (workspace and all project logs at once).

Do you have any suggestion for the UX for this?

Reply:
We could use a --full or --all flag to print logs for both the workspace and project together.

Reply:
Good idea. Let's go with `--all`